### PR TITLE
AbuseFilter and anti-spam config for WikiCanada

### PR DESCRIPTION
**Pull request by Amanda1234567**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -178,17 +178,31 @@
 			'rangeblock' => false,
 			'tag' => true,
 		),
+		'wikicanadawiki' => array(
+			'flag' => true,
+			'throttle' => true,
+			'warn' => true,
+			'disallow' = true,
```
Comment (The-Voidwalker) [inline LocalSettings.php]: missing angle bracket '`>`'

**Comment by Amanda1234567** [comment]
Fixed

File: LocalSettings.php [hunk 2]
```diff
@@ -178,17 +178,31 @@
 			'rangeblock' => false,
 			'tag' => true,
 		),
+		'wikicanadawiki' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Can't you just use rangeblock => true, and remove the rest as they're already default values?

**Comment by Amanda1234567** [comment]
@Reception123 done

**Pull request merged**